src/app/static/src/app/store/modelCalibrate/modelCalibrate.ts

```diff
@@ -59,15 +60,18 @@ export const initialModelCalibrateState = (): ModelCalibrateState => {
 };
 
 export const modelCalibrateGetters = {
-    indicators: (state: ModelCalibrateState): BarchartIndicator[] => {
-        return state.calibratePlotResult!.plottingMetadata.barchart.indicators;
+    outputFilterOptions: (state: ModelCalibrateState, getters: any, rootState: RootState) => (filterId: string) => {
+        const type = state.metadata!.filterTypes.find(f => f.id === filterId);
+        return filterAfterUseShapeRegions(type as FilterTypes, rootState).options;
```

Comment:
Put this into the metadata fetch logic, as a shallow copy of the regions in the features. Then we can remove the getter here and just use the store in the component